cmd/config_test.go

```diff
@@ -83,4 +83,17 @@ func TestConfig(t *testing.T) {
 			So(must(envClusterRoleArn()), ShouldEqual, "arn:aws:iam::127178877223:role/xacct/ops-admin")
 		})
 	})
+
+	Convey("getEnvironmentKeys", t, func() {
+		Convey("returns available environment keys", func() {
+			environments := map[string]string{
+				"dev":  "dev-kube",
+				"prod": "prod-kube",
+			}
+
+			keys := getEnvironmentKeys(environments)
+
+			So(keys, ShouldResemble, []string{"dev", "prod"})
```

Comment:
This is going to cause an issue with arbitrary order


  * /Users/jgramoll/git/truss-cli/cmd/config_test.go 
  Line 96:
  Expected: '[]string{"dev", "prod"}'
  Actual:   '[]string{"prod", "dev"}'

Reply:
D'oh, I thought `ShouldResemble` would handle that... I guess I could sort it first or something?

Reply:
Ok, sorting first should resolve that I think